packages/pools/src/concentrated/fetch-tick-data-provider.ts

```diff
@@ -256,9 +260,10 @@ export class FetchTickDataProvider implements TickDataProvider {
 
 function serializeRequest(
   response: TickDepthsResponse
-): SerializedTickDepthsResponse {
+): DeserializedTickDepthsResponse {
```

Comment:
Is this just a naming convention thing? I would think the serializeRequest has a serializeResponse